README.md

```diff
@@ -65,15 +65,15 @@ can now utilize the `babel-eslint` parser to pass in sanatized JavaScript code.
 Although this would already work by now, there will be some warnings about
 unused variables whenever you write type declarations.
 
-The `eslint` plugin `eslint-plugin-flow-vars` will mute those warnings, so let
+The `eslint` plugin `eslint-plugin-flowtype` will mute those warnings, so let
 us install all the eslint stuff we need:
 
 ```
 # ESlint stuff (you probably already did that)
 npm install eslint babel-eslint --save-dev
 
 # Now the flow related stuff
-npm install eslint-plugin-flow-vars
+npm install eslint-plugin-flowtype
```

Comment:
Ideally, this should work out of the box. I'll work with the `eslint-plugin-flowtype` collaborators to make this work with no configuration if its not already.